docs/reference/sql_support.rst

```diff
@@ -0,0 +1,195 @@
+.. versionadded:: 3.0
+
+.. _ref_sql_support:
+
+===========
+SQL support
+===========
+
+
+Connecting
+==========
+
+EdgeDB supports running SQL queries via the Postgres protocol. Any
+Postgres-compatible client can connect to your EdgeDB database by using the
+same port that is used for the EdgeDB protocol and the same database name,
+username, and password you already use for your database.
```

Comment:
We should also highlight that we only support read-only queries. No DML and DDL.

Reply:
Done. All I did here was chage "SQL queries" to "read-only SQL queries." We describe the implications of this later so I don't think detail here is necessary, but I do think it's a good idea to call it out up-front.